src/jdk.javadoc/share/man/javadoc.1

```diff
@@ -96,6 +96,22 @@ option either to recursively traverse a directory and its
 subdirectories, or to pass in an explicit list of package names.
 When you document individual source files, pass in a list of Java source
 file names.
+.SS Documentation Comments
+.PP
+The \f[V]javadoc\f[R] tool uses the documentation comment, if any, that
+immediately precedes the beginning of the declaration, whether that is
+an annotation, modifier, or the name being declared.
+If there are multiple documentation comments before the declaration,
+only the last one (closest to the declaration) will be used.
+If there are any documentation comments after the beginning of the
+declaration, they will be ignored.
+To check for any extra or misplaced documentation comments, compile your
+source code with the \f[V]javac\f[R] option \f[V]-Xlint\f[R], or more
+specifically, \f[V]-Xlint:dangling-doc-comments\f[R].
+Within a source file, you may use suppress any warnings generated by
```

Comment:
Typo?
```suggestion
Within a source file, you may suppress any warnings generated by
```

Reply:
Thanks; I'll check the underlying original.